packages/hash/frontend/src/pages/signup.page.tsx

```diff
@@ -104,11 +104,14 @@ const KratosRegistrationFlowForm: FunctionComponent = () => {
       .push(`/signup?flow=${flow.id}`, undefined, { shallow: true })
       .then(() =>
         oryKratosClient
-          .submitSelfServiceRegistrationFlow(String(flow?.id), {
-            csrf_token,
-            traits,
-            password,
-            method: "password",
+          .updateRegistrationFlow({
+            flow: String(flow?.id),
```

Comment:
Given line 104 I don't think this would be nullable:
```suggestion
            flow: flow.id,
```

Reply:
Yes, thanks!

Reply:
Applied in 1ca939b057484c10cf54f7660758dc1ce4e0cb19